lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -96,7 +96,13 @@ def emit_post_create_message
           if have_delivery_config?
             msg("Your cookbook is ready. To setup the pipeline, type `cd #{cookbook_name_or_path}`, then run `delivery init`")
           else
-            msg("Your cookbook is ready. Type `cd #{cookbook_name_or_path}` to start working.")
+            msg("Your cookbook is ready. Type `cd #{cookbook_name_or_path}` to enter it.")
+            msg("\nThere are several commands you can run to get started locally developing and testing your cookbook.")
+            msg("Type `delivery local --help` to see a full list.")
```

Comment:
Alright! So we are going to let any user play around with this new command? (just verifying from the conversation we had where we said we will be in front of the user to teach them or at least explain how it works)